webdriver/tests/support/asserts.py

```diff
@@ -1,4 +1,4 @@
-import base64
+from base64 import decodebytes
```

Comment:
`from ...` imports usually go under normal imports.

Reply:
OK, I'll try to match https://www.python.org/dev/peps/pep-0008/#imports too, especially the "Imports should be grouped in the following order" bit.